### PR TITLE
force no-color output when stdout is not a terminal

### PR DESCRIPTION
**Pull request by gaetano-guerriero**

### CODE REVIEW

File: src/eyed3/utils/console.py [hunk 1]
```diff
@@ -67,15 +67,17 @@ def __getattribute__(self, name):
...
-    def init(enabled):
-        if not enabled:
-            AnsiCodes._USE_ANSI = False
-        else:
-            AnsiCodes._USE_ANSI = True
-            if (("TERM" in os.environ and os.environ["TERM"] == "dumb") or
-                    ("OS" in os.environ and os.environ["OS"] == "Windows_NT")):
-                AnsiCodes._USE_ANSI = False
+    def _term_supports_color():
+        if (os.environ.get("TERM") == "dump" or
```
Comment (nicfit) [inline src/eyed3/utils/console.py]: "dump"?

**Comment by nicfit** [comment]
Is there a behavior difference here?

File: src/eyed3/utils/console.py (hunk 1 above)
Comment (gaetano-guerriero) [reply src/eyed3/utils/console.py]: This was a typo, it is now fixed.

**Comment by gaetano-guerriero** [comment]
```bash
$ eyed3 1.mp3 > 1.info
```
now produces output with colors in 1.info, so it is not usable. The same happens if output is sent through a pipe to another process.

With this patch colored output is always disabled when output is not a terminal.
It is disabled in those cases since `sys.stdout.isatty()` is false.

This is a common behaviour of command line utilities, for instance `ls` is colored by default while `ls | cat` is not.

**Comment by nicfit** [comment]
Ah, yea. I see it now. :) Thanks!

**Pull request merged**